main/gcc/APKBUILD

```diff
@@ -194,6 +194,7 @@ source="ftp://gcc.gnu.org/pub/gcc/releases/gcc-${_pkgbase:-$pkgver}/gcc-${_pkgba
 	300-main-gcc-add-musl-s390x-dynamic-linker.patch
 
 	fix-rs6000-pie.patch
+	gcc-ppc-pure64.patch
```

Comment:
Hi @fabled. I have updated it in my branch and consequently in this PR.
Can you please take a look?
I have also changed powerpc64-* to powerpc64le-* because although it said it was targeting ppc64le in the original commit, it did not work for that arch. Now it does.